app/src/sharedTest/java/org/oppia/app/profile/PinPasswordActivityTest.kt

```diff
@@ -77,6 +79,15 @@ class PinPasswordActivityTest {
       .inject(this)
   }
 
+  @Test
+  fun testPinPasswordActivityWithAdmin_activityVisible_checkKeyboardVisibilityByDefault() {
```

Comment:
Update test case to something more appropriate because `activityVisible` is redundant because it would be visible for sure.
Something like:
`testPinPasswordActivityWithAdmin_keyboardIsVisibleByDefault()`

Reply:
@Luffy18346 Thanks